server/db/init_db.js

```diff
@@ -1,7 +1,7 @@
 const { PostManager, UserManager } = require('./db_manager');
 const { MongoClient } = require('mongodb');
-const uri = 'mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass&ssl=false';
-const client = new MongoClient(uri);
+require('dotenv').config();
+const client = new MongoClient(process.env.URI);
```

Comment:
You may think about some config file that could set the default values or cast to the correct type all values. Maybe even log the error if you haven't set the required environment variable that hasn't a default value at all.